torchtune/models/llava_next/image_processor.py

```diff
@@ -0,0 +1,344 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
I personally liked the style that huggingface uses, where everything that is specific to the model, stays close to the model, like here: https://github.com/huggingface/transformers/tree/main/src/transformers/models/llava_next

And util functions that can be used by many models can go in more generic folders: https://github.com/huggingface/transformers/blob/v4.41.1/src/transformers/image_utils.py

happy to change if you guys want me to

Reply:
I think keeping in models makes sense since they are model specific. We can move any general utilities out into data/ or modules/. It's the same idea as how Gemma needed its own decoder, Phi3 needed its own tokenizer, and they were kept in the models folder.